#include <iostream>
using namespace std;

int main() {
    try {
        throw "My Custom Exception Kushagra";
    } catch (const char* ch) {
        cout << "Caught " <<ch << endl;
    }
}
